content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -59,41 +79,137 @@ const result = await prisma.posts.findMany({
+| <inlinecode>cat &#124; pig</inlinecode> | No     | The text doesn't contain 'cat' or 'pig' |
+| `fox <-> dog`                           | Yes    | 'dog' follows 'fox' in the text         |
+| `dog <-> fox`                           | No     | 'fox' doesn't follow 'dog' in the text  |
 
-The `search` field uses the database's native querying capabilities under the hood. This means that the Prisma Client [supports what the database supports](https://www.postgresql.org/docs/12/functions-textsearch.html).
+For the full range of supported operations, see the [PostgreSQL full text search documentation](https://www.postgresql.org/docs/12/functions-textsearch.html).
+
+### MySQL
+
+The following examples demonstrate use of the MySQL 'and' (`+`) and 'not' (`-`) operators:
+
+```ts
+// All posts that contain the words 'cat' or 'dog'.
+const result = await prisma.posts.findMany({
+  where: {
+    body: {
+      search: 'cat dog',
+    },
+  },
+})
+
+// All posts that contain the words 'cat' and not 'dog'.
+const result = await prisma.posts.findMany({
+  where: {
+    body: {
+      search: '+cat -dog',
+    },
+  },
+})
+
+// All drafts that contain the words 'cat' and 'dog'.
+const result = await prisma.posts.findMany({
+  where: {
+    status: 'Draft',
+    body: {
+      search: '+cat +dog',
+    },
+  },
+})
+```
 
 To get a sense of how the query format works, consider the following text:
 
 **"The quick brown fox jumps over the lazy dog"**
 
 Here's how the following queries would match that text:
 
-| Query                                   | Match? | Description                         |
-| :-------------------------------------- | :----- | :---------------------------------- |
-| `fox & dog`                             | Yes    | The text contains fox and dog       |
-| `dog & fox`                             | Yes    | The text contains dog and fox       |
-| `dog & cat`                             | No     | The text doesn't contain cat        |
-| `!cat`                                  | Yes    | There is not cat in the text        |
-| <inlinecode>fox &#124; cat</inlinecode> | Yes    | The text contains fox or cat        |
-| <inlinecode>cat &#124; pig</inlinecode> | No     | The text doesn't contain cat or pig |
-| `fox <-> dog`                           | Yes    | dog follows fox in the text         |
-| `dog <-> fox`                           | No     | dog doesn't follow fox in the text  |
+| Query       | Match? | Description                             |
+| :---------- | :----- | :-------------------------------------- |
+| `+fox +dog` | Yes    | The text contains 'fox' and 'dog'       |
+| `+dog +fox` | Yes    | The text contains 'dog' and 'fox'       |
+| `+dog +cat` | No     | The text contains 'dog' but not 'cat'   |
+| `-cat`      | Yes    | 'cat' is not in the text                |
+| `fox dog`   | Yes    | The text contains 'fox' or 'cat'        |
+| `-cat -pig` | No     | The text doesn't contain 'cat' or 'pig' |
```

Comment:
Sorry, I was re-reading the MySql docs while checking this document and they do support a few more operators.
They are:

1. `@distance` determines if two or more words start within a specified distance of each other.  `"cat dog" @2`
2. `> <` Adjusts the relevance of the word in the search. `+cat (>dog <pig)` find rows that contain cat and dog or cat and pig. Rank the ones with `cat dog` higher than `cat pig`
3. `( )` group into subexpressions. `+cat (fox dog)` find rows that have either `cat fox` or `cat dog` 
4. `~`causes a works contribution to the rows relevance to be negative. `+cat (dog ~fox)` find rows with `cat dog` or `cat fox` but rank the ones containing `fox` much lower. 
5. `"` match the exact phrase `"cat food"` row must contain the full phrase to match
The docs also have some extra helpful examples

Reply:
I've added in some extra examples to illustrate these - please have a check that they look correct!